test/membrane/internal-slot-protection.spec.js

```diff
@@ -9,7 +9,10 @@ describe('membrane', () => {
             expect(value).toBe('import');
             expect(this instanceof HTMLLinkElement).toBeTrue();
         });
```

Comment:
You'll need to rebase to fix this test. While I was reviewing I discovered that the `distortionMap` was originally created incorrectly.

Reply:
rebased!